consensus/types/src/execution_payload.rs

```diff
@@ -56,4 +57,15 @@ impl<T: EthSpec> ExecutionPayload<T> {
             transactions: VariableList::empty(),
         }
     }
+
+    #[allow(clippy::integer_arithmetic)]
+    /// Returns the maximum size of an execution payload.
+    pub fn max_execution_payload_size() -> usize {
+        // Fixed part
+        Self::empty().as_ssz_bytes().len()
+        // length of List * size_of(uint8)
+        + (T::max_extra_data_bytes() * <u8 as Encode>::ssz_fixed_len())
+        // length of List * offset size * max size of transaction
+        + (T::max_transactions_per_payload() *ssz::BYTES_PER_LENGTH_OFFSET * T::max_bytes_per_transaction())
```

Comment:
Are you sure this is correct. 
Perhaps:
```rust
(T::max_transactions_per_payload() * (ssz::BYTES_PER_LENGTH_OFFSET + T::max_bytes_per_transaction()))
```

Reply:
Sorry had already fixed this up in b16ca82 but forgot to push the changes.